.github/workflows/update-bff-dependency/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "Update BFF dependency"
```

Comment:
Can you rename the directory with a name different from the action file? for example update-dependency